zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/JobControllerTest.java

```diff
@@ -485,4 +494,64 @@ void shouldRejectUpdateJobNoBody() {
         .expectBody()
         .json(expectedBody);
   }
+
+  @ParameterizedTest
+  @MethodSource("tenantLists")
+  void shouldRejectActivateJobWithUnauthorizedTenantWhenMultiTenancyEnabled(
+      final List<String> tenantIds) {
+    // given
+    when(multiTenancyCfg.isEnabled()).thenReturn(true);
+    final var request =
+        """
+        {
+          "type": "TEST",
+          "maxJobsToActivate": 2,
+          "requestTimeout": 100,
+          "timeout": 100,
+          "fetchVariable": [],
+          "tenantIds": %s,
+          "worker": "bar"
+        }"""
+            .formatted(tenantIds.stream().map("\"%s\""::formatted).toList());
```

Comment:
😂 I love this. I was convinced there must be an easier way but I can't find any.

Reply:
I tried other things. This was by far the best I could come up with 😆 
I could provide the strings I need with escaped stuff as an alternative, but the verification in the assertions becomes A LOT uglier then (because we want to determine if it's one or multiple tenants). So, I decided for this to stay 🤷